media/themes/modern.msstyles/textfiles/themes.INI

```diff
@@ -0,0 +1,108 @@
+;--------------------------------------------
+; file: textfiles\themes.ini
+;--------------------------------------------
+
+[documentation]
+DisplayName = Modern Style	
+LastUpdated = 08/28/06
+ToolTip = Visual Style
+Author = Microsoft Design Team
+Company = Microsoft Corporation
```

Comment:
This... is a bit questionable.

Reply:
its just a placeholder file it bears no relationship and is not built into reactos. nevertheless i have already changed this and other things in a patch i was submitting later

Reply:
( unless you are talking explicitly about the author= and company= lines which are objectively incorrect, i just missed these lines )

Reply:
How did these fields even get here is the question.

Reply:
these are required fields in a themes.INI file as per all other themes

Reply:
as stated, this is just a placeholder file I left in. it will be removed in the next patch

Reply:
ok cernodile was surprised you used "microsoft" as placeholder caption, instead of using "reactos" in the first place.

Reply:
the file has nothing to do with this patch, i just copied it there without realising